src/pages/Gallery/components/Category/index.tsx

```diff
@@ -0,0 +1,34 @@
+import * as S from './category.styles'
+
+import React, { useState } from 'react'
+
+type CategoryProps = {
+  setSelectedCategory: (category: string) => void
```

Comment:
```suggestion
  setSelectedCategory: Dispatch<SetStateAction<string>>
```

상태를 업데이트하는 함수(set함수)는 위처럼 명시적으로 알려줄 수 있는 타입을 사용하는 게 좋슴미당 !